clippy_lints/src/methods.rs

```diff
@@ -834,6 +879,39 @@ fn lint_filter_next(cx: &LateContext, expr: &hir::Expr, filter_args: &MethodArgs
     }
 }
 
+// Type of MethodArgs is potentially a Vec
+/// lint use of `filter().map() for Iterators`
```

Comment:
Small nit: `` `filter().map() for Iterators` `` → `` `filter().map()` for `Iterators` ``